src/reaction.jl

```diff
@@ -85,9 +92,6 @@ function get_netstoich(subs, prods, sstoich, pstoich)
     [el for el in nsdict if !_iszero(el[2])]
 end
 
-# Get the net stoichiometries' type.
-netstoich_stoichtype(::Vector{Pair{S, T}}) where {S, T} = T
```

Comment:
No longer needed as we need to check both `S` and `T` (as `S` might now need to be converted from `Num` to `BasicSymbolic{Real}`.